website/files/models/base.py

```diff
@@ -91,6 +91,19 @@ def restore(self, recursive=True, parent=None):
         TrashedFileNode.remove_one(self)
         return restored
 
+    def get_guid(self, create=False):
+        """Attempt to find a Guid that points to this object.
+        One will be created if requested.
+        :rtype: Guid
+        """
+        try:
+            # Note sometimes multiple GUIDs can exist for
+            # a single object. Just go with the first one
+            return Guid.find(Q('referent', 'eq', self))[0]
```

Comment:
@sloria @erinspace  Ha! fitcosz is an auth used for waterbutler testing. Sorry for any confusion.